src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -359,6 +360,92 @@ public void validateNoKeyPersonnelDuplicates(DataAccessRequestData darData) {
+        darData.getInternalCollaborators().stream().map(Collaborator::getEmail).toList();
+    List<String> labStaffEmails =
+        darData.getLabCollaborators().stream().map(Collaborator::getEmail).toList();
+
+    List<String> invalidMembers = new ArrayList<>();
+
+    if (emailDoesNotMatchInstitution(submitterInstitution, piEmail)) {
+      invalidMembers.add("Principal Investigator: " + piEmail);
+    }
+    if (emailDoesNotMatchInstitution(submitterInstitution, soEmail)) {
+      invalidMembers.add("Signing Official: " + soEmail);
+    }
+
+    if (emailDoesNotMatchInstitution(submitterInstitution, itEmail)) {
+      invalidMembers.add("IT Director: " + itEmail);
+    }
+
+    getErrorSummary(
+            collaboratorsEmails,
+            submitterInstitution,
+            "Internal Collaborator member: ",
+            "Internal Collaborator members: ")
+        .ifPresent(invalidMembers::add);
+
+    getErrorSummary(
+            labStaffEmails, submitterInstitution, "Lab staff member: ", "Lab staff members: ")
+        .ifPresent(invalidMembers::add);
+
+    if (!invalidMembers.isEmpty()) {
+      throw new IllegalArgumentException(
+          "All listed personnel must share the same institutional affiliation.  The following list of roles and members must have email addresses associated with your institution: "
+              + String.join(", ", invalidMembers));
+    }
+  }
+
+  private Optional<String> getErrorSummary(
+      List<String> emails,
+      Institution institution,
+      String categorySingular,
+      String categoryPlural) {
+    List<String> errors = findEmailAddressesNotInInstitution(emails, institution);
+    if (!errors.isEmpty()) {
+      return Optional.of(buildSingleErrorFromErrorList(errors, categorySingular, categoryPlural));
+    }
+    return Optional.empty();
+  }
+
+  private List<String> findEmailAddressesNotInInstitution(
+      List<String> emailAddresses, Institution institution) {
+    ArrayList<String> errors = new ArrayList<>();
+    emailAddresses.forEach(
+        collaborator -> {
+          if (emailDoesNotMatchInstitution(institution, collaborator)) {
+            errors.add(collaborator);
+          }
+        });
+    return errors;
+  }
+
+  private String buildSingleErrorFromErrorList(
+      List<String> errors, String categorySingular, String categoryPlural) {
+    StringBuilder msg = new StringBuilder();
+    if (errors.size() == 1) {
+      msg.append(categorySingular);
+    } else if (errors.size() > 1) {
+      msg.append(categoryPlural);
+    }
+    msg.append(String.join(", ", errors));
+    return msg.toString();
+  }
+
+  private boolean emailDoesNotMatchInstitution(Institution institution, String email) {
+    Institution foundInstitution = institutionService.findInstitutionForEmail(email);
+    if (foundInstitution == null || institution == null) {
+      return true;
+    }
+    return !institution.getId().equals(foundInstitution.getId());
```

Comment:
This is the same as comparing `Institution`
```suggestion
    return !institution.equals(foundInstitution);
```